client/question/question.js

```diff
@@ -633,6 +633,17 @@ tie.constant('SECONDS_TO_MILLISECONDS', SECONDS_TO_MILLISECONDS);
  * @constant
  */
 tie.constant('DEFAULT_AUTOSAVE_SECONDS', 5);
+
+/**
+ * Default time in seconds between calls to SendEventBatch.
+ *
+ * @type {number}
+ * @constant
+ */
+var DEFAULT_EVENT_BATCH_PERIOD_SECONDS = 30;
```

Comment:
Why declare this? Just put 30 in line 645 -- don't want to add stuff in global scope.

Reply:
Linter.

Reply:
Er, no -- we don't have that issue in the above line (DEFAULT_AUTOSAVE_SECONDS).

Reply:
That's correct -- the linter ignores magic numbers up to and including 5. :)

Somewhere in the file there's another example for 1000.

Reply:
Ah, I see, thanks for clarifying! It's fine to add those specific numbers (30, 1000) to [.eslintrc](https://github.com/google/tie/blob/master/.eslintrc.json#L103). I think it's better to do that than to introduce global vars.

Reply:
I see. The problem is that it changes it for the entire codebase, rather than the specific example, which can cause problems down the line. :\ 

Do you know of, like, a happy medium?

Reply:
Yes, you can disable the linter for specific lines, just like python pragmas. See e.g. https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments (6th example).

Reply:
(I suggest doing the one with the comment on the right of the line, since that's similar to how we'd do it for python.)

Reply:
I did it above to avoid any line-length issues.